Project-I-Social-Card/social-card/src/components/SocialLink/SocialLink.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+function SocialLink(props) {
+  let count = 0;
+  const handleClick = () => {
+    count++;
+    document.getElementById(props.id).textContent = count;
+  }
```

Comment:
clever way of making this work in a functional component, and this is a great precursor to utilizing `state` in a class component to hold your count data, and updating it with `this.setState` will trigger a re-render of the component without having to fetch anything from the DOM